dts/bindings/interrupt-controller/irq.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __DT_BINDING_IRQ_H
+#define __DT_BINDING_IRQ_H
+
+#if defined(CONFIG_IOAPIC) || defined(CONFIG_MVIC)
+#define	LEVEL		0x00008000
+#define	EDGE		0x00000000
+#define	LOW		0x00002000
+#define	HIGH		0x00000000
+#define EDGE_RISING	0x00
+#define EDGE_FALLING	0x00002000
+#define LEVEL_HIGH	0x00008000
+#define LEVEL_LOW	0x0000A000
```

Comment:
Can we have this defined in terms of LEVEL, EDGE, LOW, HIGH above?

ie #define LEVEL_LOW (LEVEL | LOW)

Reply:
I had tried doing like that but i am getting parsing error . so i had done other way